make.go

```diff
@@ -307,7 +307,7 @@ func (u *upstream) findDependencies(gopath, repo string) error {
 	return nil
 }
 
-func makeUpstreamSourceTarball(repo, revision string, forcePrerelease bool) (*upstream, error) {
+func makeUpstreamSourceTarball(repo, revision, tag string, forcePrerelease bool) (*upstream, error) {
```

Comment:
shouldn't that be
```
func makeUpstreamSourceTarball(repo, revision string, tag, forcePrerelease bool) (*upstream, error) {
```
instead ?

Reply:
That's a better idea! 

So this will be a boolean parameter to indicate that we want to force the tag being used, and not let dh-make-golang determinate the tag from the git refspec.

I will update my PR.